confluent_kafka/error.py

```diff
@@ -15,11 +15,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+from confluent_kafka.cimpl import KafkaException, KafkaError
 
 
-class ConsumeError(Exception):
-    __slots__ = ['message', 'reason', 'error_code']
-
+class ConsumeError(KafkaException):
```

Comment:
In an ideal world, would the naming `KafkaException` -> `KafkaError` and `KafkaError` -> `Error` make the most sense?

Reply:
Yeah unfortunately we  don't have that luxury today as its technically a breaking change. I'd like to change that or come up with a better name in the future  though.